scripts/pi-hole/php/database.php

```diff
@@ -54,4 +54,90 @@ function SQLite3_connect($filename, $mode=SQLITE3_OPEN_READONLY)
 	}
 	return $db;
 }
+
+
+/**
+ * Add domains to a given table
+ *
+ * @param $db object The SQLite3 database connection object
+ * @param $table string The target table
+ * @param $domains array Array of domains (strings) to be added to the table
+ * @param $wildcardstyle boolean Whether to format the input domains in legacy wildcard notation
+ * @return string Success/error and number of processed domains
+ */
+function add_to_table($db, $table, $domains, $wildcardstyle=false)
+{
+	// Prepare SQLite statememt
+	$stmt = $db->prepare("INSERT OR IGNORE INTO ".$table." (domain) VALUES (:domain);");
+
+	// Return early if we prepare the SQLite statement
+	if(!$stmt)
+	{
+		echo "Failed to prepare statement for ".$table." table.";
+		return "Error, added: 0\n";
```

Comment:
Is the newline at the end necessary?

Reply:
When we black-/whitelist from the audit page, we add to both, the respective list *and* the audit list at the same time. Without this newline, the outputs from the two calls would appear on the same line.

Reply:
The front-end should handles this instead.